conf/modules.config

```diff
@@ -65,6 +65,16 @@ process {
         ].join(' ').trim() }
     }
 
+    withName: TMB_CALCULATE  {
+        ext.args = { [
+            (params.min_af)		    ? "--min_AF ${params.min_af}"   : "--min_AF 0",
```

Comment:
The `: "--min_AF 0",` should **not** be set, as the default value is set in `nextflow.config`. Replace it with `: "",`. This avoids conflict, when the default value is changed in `nextflow.config` The same applies to line 71-74 as well.